.ci/setup_env_alinux.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Ant Group
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+cidir=$(dirname "$0")
+source "/etc/os-release" || "source /usr/lib/os-release"
+source "${cidir}/lib.sh"
+
+# Obtain AlibabaCloud Linux version
+if [ -f /etc/os-release ]; then
+  alinux_version=$(grep VERSION_ID /etc/os-release | cut -d '"' -f2)
+else
+  alinux_version=$(grep VERSION_ID /usr/lib/os-release | cut -d '"' -f2)
+fi
+
+[ "$alinux_version" == 3 ] || die "This script is for alinux 3 only"
+
+# Send error when a package is not available in the repositories
+echo "skip_missing_names_on_install=0" | sudo tee -a /etc/yum.conf
+
+# Ensure EPEL repository is configured
+sudo -E dnf -y install epel-release
+
+# Enable priority to AlibabaCloud Linux Base repo in order to
+# avoid perl updating issues
+if [ -f "/etc/yum.repos.d/AliYun.repo" ]; then
+	repo_file="/etc/yum.repos.d/AliYun.repo"
+fi
+
+[ -n "${repo_file:-}" ] || die "Unable to find the AlibabaCloud Linux base repository file"
+
+if [ "$(tail -1 ${repo_file} | tr -d '\n')" != "priority=1" ]; then
```

Comment:
Assume the last line is `priority=something` is a little weak ans so error-prone IMHO.

Reply:
It's hard to check should append `priority=...`., but the image that the repo file seldom be updated manually, so this is the easiest way to avoid this issue :
https://github.com/kata-containers/tests/issues/4535

Reply:
ok, got it. thx.